libs/execution_context/ebpf_protocol.h

```diff
@@ -332,3 +333,15 @@ typedef struct _ebpf_operation_get_next_id_reply
     struct _ebpf_operation_header header;
     ebpf_id_t next_id;
 } ebpf_operation_get_next_id_reply_t;
+
+typedef struct _ebpf_operation_get_object_info_request
+{
+    struct _ebpf_operation_header header;
+    uint64_t handle;
```

Comment:
Why not pass object_type here and avoid trial and error in ebpf_core handler?

Reply:
because the caller doesn't specify it in the libbpf API and trial and error here is more efficient than trial and error from user mode with a separate ioctl per trial.   Libbpf just takes an fd and on Linux calls the BPF_OBJ_GET_INFO_BY_FD syscall which also takes an untyped fd.

Reply:
n more trial and error needed, as Anurag pointed out